src/Microsoft.ML.AutoML/Experiment/MetricsAgents/RankingMetricsAgent.cs

```diff
@@ -57,9 +57,9 @@ public bool IsModelPerfect(double score)
             }
         }
 
-        public RankingMetrics EvaluateMetrics(IDataView data, string labelColumn)
+        public RankingMetrics EvaluateMetrics(IDataView data, string labelColumn, string groupIdColumn)
```

Comment:
Can we just put groupId column as an option when creating RankingMetricsAgent? So that we don't have to add an extra groupIdColumn to IAgent.

Reply:
Yes, we can do that. I think a possible concern is that GroupId is required by RankingTrainers, and passing it in as an option gives me the impression that it's optional. I think that if we make the default value of `GroupId`  "GroupId" and document that, it might be okay. What you do think?